stage0/src/lib.rs

```diff
@@ -252,8 +258,12 @@ pub fn rust64_start(encrypted: u64) -> ! {
         entry = VirtAddr::new(header.e_entry);
     }
 
-    let rsdp = acpi::build_acpi_tables(&mut fwcfg).unwrap();
+    let mut acpi_digest = Sha256::default();
```

Comment:
Why are ACPI tables measured separately?

Reply:
Stage0 constructs ACPI tables by reading multiple files and command over the fw_cfg device from the VMM. The number of files and commands can change, so this keeps extending the single digest for all the files and commands.